src/plugins/home/public/application/components/_home.scss

```diff
@@ -1,8 +1,10 @@
+@import '../../../../../core/public/mixins';
+
 .homWrapper {
+  @include kibanaFullBodyMinHeight();
   background-color: $euiColorEmptyShade;
   display: flex;
   flex-direction: column;
-  min-height: calc(100vh - #{$euiHeaderHeightCompensation});
 }
```

Comment:
I went with explicit imports because this is imho the way to go. But if @elastic/kibana-design thinks it's better to have these mixins in `src/core/public/core_app/styles/_mixins.scss` instead and use implicit imports, I can change it.

Reply:
I think it's good that we exercise caution before adding to the public/core_app/styles/ directory since that stuff all gets built and prepended to .scss imports, but the stuff being added by this PR feels appropriate given it's global nature.

If I'm following correctly, this means we could move the _core.scss and _variables.scss files and combine the _mixins.scss files from `.../public/` to `.../public/core_app/styles/` and remove all the direct imports? I think the import of the header variables atop the _variables.scss file could also then be removed.

Let me know if I'm missing other pitfalls in doing this, but it seems like a cleaner way to go and I sense it will reduce the likelihood of redundant variable imports and such.

Reply:
Any thoughts on this? Overall, a minor item that I wouldn't block merging on.

Reply:
Sorry, I missed that one. 

> If I'm following correctly, this means we could move the _core.scss and _variables.scss files and combine the _mixins.scss files from .../public/ to .../public/core_app/styles/ and remove all the direct imports? 

Yea. I need to confirm the consequences of doing that regarding core internal scss imports / hierarchy, but that would be the idea.

I will take a shot at it, and if it's too complicated, I will do it during phase 2